frontend/packages/dev-console/src/components/pipelines/PipelineDetailsPage.tsx

```diff
@@ -42,7 +44,9 @@ const PipelineDetailsPage: React.FC<DetailsPageProps> = (props) => {
       .catch((error) => setErrorCode(error.response.status));
   }, [name, namespace]);
 
-  const augmentedMenuActions: KebabAction[] = useMenuActionsWithUserLabel(menuActions);
+  const augmentedMenuActions: KebabAction[] = useMenuActionsWithUserLabel(
+    getPipelineKebabActions(latestPipelineRun, !_.isEmpty(templateNames)),
```

Comment:
Nit: No need to use lodash.
```suggestion
    getPipelineKebabActions(latestPipelineRun, templateNames.length > 0),
```